src/error.js

```diff
@@ -1,11 +1,24 @@
 /* Error handling.
  */
-import { get } from 'lodash';
 import {
+    GATEWAY_TIMEOUT,
     INTERNAL_SERVER_ERROR,
     NOT_FOUND,
-    GATEWAY_TIMEOUT,
 } from 'http-status-codes';
+import { get, omitBy } from 'lodash';
+
+// We are accidentally leaking auth headers to loggly when throwing
+// an OpenAPIError. We want to fix this properly in the long term,
+// but as a short term solution we are simply going to omit the headers
+// from the error here. This operation is case insensitive.
+function pruneAuthorizationHeaders(headers) {
+    if (!headers) {
```

Comment:
We need this because omitBy will return an empty object if headers is null, and there are some test assertions expecting headers to be null... I didn't delve too deep into why/if that matters.

Reply:
¯\_(ツ)_/¯ fair enough keeping with previous behavior, bet we can let whoever does the TS conversion figure out why it matters now. 😆